README.md

```diff
@@ -95,7 +95,7 @@ subnet, you can provide an `AllowList` configuration in your genesis file:
       "targetBlockRate": 2,
       "blockGasCostStep": 500000
     },
-    "allowListConfig": {
+    "contractDeployerAllowListConfig": {
```

Comment:
based on the config, I don't think this is correct. You are missing the inner `allowListConfig` nesting.

Reply:
This works as is since the allow list config is embedded in the contract deployer allow list config.